File: frontend/react-app/src/controller/SupportController.ts
import api from "../http";
import {ICreateSupportMessage} from "../model/support/ICreateSupportMessage";

export class SupportController {

    public static async addSupportMessage(createSupportMessage:ICreateSupportMessage){
       return  api.post<ICreateSupportMessage>('api/v1/support/add/client/help-message',createSupportMessage).then(r=>r.data);
    }
}

File: backend/src/main/java/app/miniappspring/controller/SupportController.java
package app.miniappspring.controller;

import app.miniappspring.dto.support.CreateSupportMessageDto;
import app.miniappspring.dto.support.SupportMessageDto;
import app.miniappspring.service.SupportService;
import lombok.RequiredArgsConstructor;
import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("api/v1/support")
@RequiredArgsConstructor
public class SupportController {

    private final SupportService supportService;

    @PostMapping("/add/client/help-message")
    public SupportMessageDto addMessageUserOboutHelp(@RequestBody CreateSupportMessageDto createSupportMessageDto) {
        return supportService.addMessageUserOboutHelp(createSupportMessageDto);
    }
}
